lib/govuk_schemas/random_example.rb

```diff
@@ -29,9 +29,9 @@ def initialize(schema:)
     #
     # @param schema_key_value [Hash]
     # @return [GovukSchemas::RandomExample]
-    def self.for_schema(schema_key_value)
+    def self.for_schema(schema_key_value, &block)
       schema = GovukSchemas::Schema.find(schema_key_value)
-      GovukSchemas::RandomExample.new(schema: schema)
+      GovukSchemas::RandomExample.new(schema: schema).payload(&block)
     end
 
     # Return a content item merged with a hash and with the excluded fields removed.
```

Comment:
This description isn't accurate any more. It should probably describe the use case without a block first and then talk about passing a block.

Reply:
updated